src/iort_lib/iort.cpp

```diff
@@ -16,11 +18,34 @@ using namespace std::chrono_literals;
 #include <deque>
 #endif
 
+std::string topic;
+struct mosquitto *mosq;
+bool new_msg = false;
+Json::Value payload;
+
+void on_connect(struct mosquitto *mosq, void *obj, int rc) {
+    if (rc) {
+        exit(-1);
+    }
+    mosquitto_subscribe(mosq, NULL, topic.c_str(), 0);
+}
+
+void on_message(struct mosquitto *mosq, void *obj, const struct mosquitto_message *msg) {
+#ifdef DEBUG
+    std::cout << "New message with topic " << msg->topic << " " << (char *) msg->payload << std::endl;
+#endif
+    Json::Reader reader;
+    reader.parse((char *) msg->payload, payload);
+    new_msg = true;
+}
+
 namespace iort
 {
 static const std::string FUNCTION_URL =
     "https://5p1y6wnp3k.execute-api.us-west-2.amazonaws.com/default/"
     "iort_lib_query";
+static const std::string ENDPOINT_URL =
+             "a22ztr6so9o7g8-ats.iot.us-west-2.amazonaws.com";
```

Comment:
If you have the time it'd be nice to be able to read these from the command line or from a config file. Totally fine to leave it as is though.